app/scripts/TiledPixiTrack.js

```diff
@@ -1055,13 +1053,13 @@ class TiledPixiTrack extends PixiTrack {
         // .base(Math.E)
         .domain([offsetValue, maxValue + offsetValue])
         // .domain([offsetValue, this.maxValue()])
-        .range([minDimension, maxDimension]);
+        .range([maxDimension, minDimension]);
```

Comment:
Interesting these are flipped

Reply:
Yeah, not sure what's going on there. Looking above it looks `minDimension` and `maxDimension` are exactly the same.

Reply:
It looks like both `[minDimension, maxDimension]` and `[maxDimension, minDimension]` where introduced in https://github.com/higlass/higlass/commit/14665322b56f6acbbbde6a32ef9eb77c11659197.

Reply:
It doesn't matter to me and I don't really know what this function is doing, so happy to merge.

Reply:
Do some of these resgen overwrites remove patches I sent in?

Reply:
These changes are really to try to understand and consolidate some core differences between resglass and higlass.

I haven't taken a close look at the diffs between this and your changes and am happy to postpone until we get those merged.